safe/impact_function/postprocessors.py

```diff
@@ -269,6 +269,36 @@ def run_single_post_processor(layer, post_processor):
     return True, None
 
 
+def should_run(keywords, post_processor):
+    exposure = keywords['exposure_keywords']['exposure']
+    hazard = keywords['hazard_keywords']['hazard']
+    try:
+        run_filter = post_processor['run_filter']
+    except KeyError:
+        # if no run_filter is defined we run the postprocessor
+        return True, None
+
+    msg = 'Postprocessor "{name}" did not run because hazard ' \
```

Comment:
Will this message be shown to the user? If so, we make it translatable.

Reply:
Add avoid to use `\` if possible

Reply:
done